scripts/components/timer.js

```diff
@@ -28,6 +40,11 @@ var TimerPanel = module.exports = React.createClass({
         var task = session.get('task');
         var duration = task.get('duration');
         var mins = Math.floor(duration / (60 * 1000));
+        window.setTimeout(function() {
+            var input = document.querySelector('.gotr-selector-box-input');
+            input.focus();
+        }, 500);
+
```

Comment:
This should be achievable by adding the `autofocus` attribute to the input box in the selector component. If that fails to work then you can also try adding a `componentDidMount` handler to the selector component, and using React references to focus the input box.

```js
componentDidMount: function() {
  this.refs.minsInput.getDOMNode().focus();
}
```